Bug: https://bugs.python.org/issue27369
From ffd1712c05558e8a622c03a63e43e295afa1ddfc Mon Sep 17 00:00:00 2001
From: Sebastian Pipping <sebastian@pipping.org>
Date: Wed, 22 Jun 2016 19:40:14 +0200
Subject: [PATCH] Fix Python 3.3.5/3.4.4/3.5.1 tests for Expat 2.2.0

Related commit in Expat: 550eb6bbaa7e6efc44e94cab945fe9c781c01211
---
 Lib/test/test_pyexpat.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Lib/test/test_pyexpat.py b/Lib/test/test_pyexpat.py
index 216a46b..9e9d665 100644
--- a/Lib/test/test_pyexpat.py
+++ b/Lib/test/test_pyexpat.py
@@ -666,7 +666,10 @@ class MalformedInputTest(unittest.TestCase):
             parser.Parse(xml, True)
             self.fail()
         except expat.ExpatError as e:
-            self.assertEqual(str(e), 'XML declaration not well-formed: line 1, column 14')
+            self.assertIn(str(e), [
+                    'XML declaration not well-formed: line 1, column 13',
+                    'XML declaration not well-formed: line 1, column 14',
+                    ])
 
 class ErrorMessageTest(unittest.TestCase):
     def test_codes(self):
-- 
2.9.0

